pkgs/development/python-modules/sourmash/default.nix

```diff
@@ -20,24 +21,29 @@
 
 buildPythonPackage rec {
   pname = "sourmash";
-  version = "4.8.4";
+  version = "4.8.7";
   format = "pyproject";
-  disabled = pythonOlder "3.8";
+  disabled = pythonOlder "3.9";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-Q1hMESwzEHGXcd4XW4nLqU8cLTCxrqRgAOr1qB77roo=";
+    hash = "sha256-DNGOpJ2UzpL1yFsjN897cyjvMBPDMxFBTbcbKZYqk/0=";
   };
 
   cargoDeps = rustPlatform.fetchCargoTarball {
     inherit src;
     name = "${pname}-${version}";
-    hash = "sha256-HisWvJgx15OfYoMzzqYm1JyY1/jmGXBSZZmuNaKTDjI=";
+    hash = "sha256-MY6KrXmFAlL3RK7BFNQrte4XiNdLisSkVh035MfTy+A=";
   };
 
+  preConfigure = lib.optionalString stdenv.isDarwin ''
+    export MACOSX_DEPLOYMENT_TARGET=10.14
```

Comment:
Can we get this version from some darwin package or so?

Reply:
I moved this up to `pkgs/top-level/python-packages.nix`, and used the `overrideSDK` method to declare the min version:
```
  sourmash = callPackage ../development/python-modules/sourmash {
    stdenv = if pkgs.stdenv.isDarwin then pkgs.overrideSDK pkgs.stdenv {
      darwinMinVersion = "10.14";
    } else pkgs.stdenv;
  };
```

Hopefully this is better and less prone to errors in the future